chromaui/archive-storybook
# Use peer dependencies to allow using a range of SB versions

## What Changed

Rather than use a direct depedency, use a peer dep to get SB packages we need. This allows us to be installed into projects that already contain Storybook and mean we won't get version conflicts.

NOTE: this only supports 7.0 at this stage. We'd have to use a different `main.js` file and change the bin scripts if we wanted to support 6.5. I'm not sure if that effort is justified at this stage in the release cycle for 7.x

## How to test

Install (canary) package into project both with and without existing SB install, with instructions from here: https://github.com/chromaui/test-archiver/pull/6

Check archive storybook runs + builds without errors.


## Change Type





<details>
  <summary>📦 Published PR as canary version: <code>0.0.7--canary.3.df51966.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @chromaui/archive-storybook@0.0.7--canary.3.df51966.0
  # or 
  yarn add @chromaui/archive-storybook@0.0.7--canary.3.df51966.0
  ```
</details>
